src/tests/functional/plugin/shared/include/behavior/executable_network/exec_network_base.hpp

```diff
@@ -298,6 +296,16 @@ TEST_P(ExecutableNetworkBaseTest, pluginDoesNotChangeOriginalNetwork) {
     compare_functions(cnnNet.getFunction(), referenceNetwork);
 }
 
+TEST_P(AutoMultiExecutableNetworkBaseTest, canLoadCorrectNetworkToGetMetricAndCheckConfig) {
+    auto execNet = ie->LoadNetwork(cnnNet, target_device, configuration);
+    for (const auto& configItem : configuration) {
+        InferenceEngine::Parameter param;
+        ASSERT_NO_THROW(param = execNet.GetMetric(configItem.first));
+        ASSERT_FALSE(param.empty());
+        ASSERT_EQ(param, InferenceEngine::Parameter(configItem.second));
```

Comment:
what is auto plugin specific here?

Reply:
So far AUTO ExecutableNetwork only has RO properties. 

So AUTO ExecutableNetwork does not support GetConfig() yet.

Reply:
The similar behavior we have for other HW plugins